src/test/java/org/osiam/client/AbstractIntegrationTestBase.java

```diff
@@ -50,8 +50,8 @@
     protected static final Client CLIENT = ClientBuilder.newBuilder().register(JacksonFeature.class).build();
 
     static {
-        OsiamConnector.setConnectTimeout(10000);
-        OsiamConnector.setReadTimeout(30000);
+        OsiamConnector.setConnectTimeout(Integer.parseInt(System.getProperty("debug.timeout.connect", "-1")));
+        OsiamConnector.setReadTimeout(Integer.parseInt(System.getProperty("debug.timeout.read", "-1")));
```

Comment:
Please set the default values like before, but in this case 10000 for the read timeout is enough.